### DISCUSSION TITLE
changeAccount not triggered on coinbase when enable both metamask and coinbase on browser

### DISCUSSION THREAD

**Discussion by trhung26620** [wevm/wagmi#2845] [Q&A]
can someone with experience connecting wallets with wagmi help me? I see the demo example on https://wagmi.sh/examples/connect-wallet not working properly in case both metamask and coinbase wallet are installed on user browser, if user connect to coinbase wallet it will show address only for now, however if user changes address on coinbase extension, the address is not updated again, I observe this issue will be resolved by disabling metamask. Can someone tell me how to update the coinbase address when the user changes while the metamask is still enabled?

**Comment by gope-gope** [comment]
I am experiencing the same issue, exactly as you described it.

**Reply by trhung26620** [reply]
Please tell me if you resolved

**Reply by frantrick** [reply]
Hello guys anybody solved this?
